Assets/Scripts/CountDownTimer.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class CountDownTimer : MonoBehaviour
+{
+    public float timeValue;
+    public Text texTimer;
+    // Start is called before the first frame update
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update(){
+        if (timeValue > 0)
+        {
+            timeValue -= Time.deltaTime;
+        }
+        else {
+            timeValue = 0;
+            SceneManager.LoadScene("GameOverScene");
+        }
+        DisplayCountDownTime(timeValue);
+
+    }
+
+    void DisplayCountDownTime(float DisplayTheTime) {
+
+        if (DisplayTheTime < 0) {
+            DisplayTheTime = 0;
+        } else if(DisplayTheTime > 0){
```

Comment:
add space after else if and before curly braces:
`else if (DisplayTheTime > 0) {`